src/iop/bilateral.cc

```diff
@@ -92,6 +92,16 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+const char *description(struct dt_iop_module_t *self)
+{
+  return dt_iop_set_description(self, _("denoise high ISO images. this can be used to blur surfaces\n"
+                                        "while preserving edges."),
+                                      _("corrective and creative"),
```

Comment:
corrective only ?

Reply:
"apply edge-aware surface blur to denoise or smoothen textures"